argoverse/evaluation/detection/utils.py

```diff
@@ -333,10 +329,17 @@ def rank(dts: List[ObjectLabelRecord]) -> Tuple[np.ndarray, np.ndarray]:
         ranks: Ranking for the detections (N,).
         scores: Detection scores (N,).
     """
-    scores = np.array([dt.score for dt in dts])
+    scores = np.array([dt.score for dt in dts.tolist()])
     ranks = scores.argsort()[::-1]
+
     ranked_detections = dts[ranks]
-    return ranked_detections, scores[:, np.newaxis]
+    ranked_scores = scores[ranks, None]
```

Comment:
I think `scores[ranks, None]` will not agree with the docstring, which says `(N,)` for scores?

Reply:
won't it be (N,1)?

Reply:
maybe we need to update the names of the return variables, in the docstring as well?